client/src/components/Settings/index.tsx

```diff
@@ -10,46 +10,125 @@ import chevron from "icons/chevron.svg";
 import "./index.css";
 import DropDown from "components/Dropdown";
 import Text from "components/Text";
+import styled from "styled-components";
 
 const engineArray: Engine[] = ["youtube", "soundcloud"];
 
+const Button = styled.button`
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  height: 2rem;
+  will-change: transform;
+  border: none;
+  text-decoration: none;
+  outline: none;
+  background-color: #393838;
```

Comment:
Already commented this, let's pull the colors (all around) from the theme by func interpolation


```suggestion
  background-color: ${( {theme} ) => theme.foo.bar};
```